assets/cluster-bootstrap/podsecurity-alert.yaml

```diff
@@ -0,0 +1,25 @@
+# Source: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.11/bindata/assets/alerts/podsecurity-violations.yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: podsecurity
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+  - name: pod-security-violation
+    rules:
+    - alert: PodSecurityViolation
+      annotations:
+        summary: One or more workloads users created in the cluster don't match their Pod Security profile
+        description: >-
+          A workload (pod, deployment, deamonset, ...) was created somewhere in the cluster but it
+          did not match the PodSecurity "{{ `$labels.policy_level` }}" profile defined by its namespace either via the cluster-wide
```

Comment:
Don't know if matters, but this has:
```
"{{ `$labels.policy_level` }}"
```
While the source file has:
```
"{{ $labels.policy_level }}"
```
i.e. differs in backquotes

Reply:
Right, that is required in order to generate the proper bindata.

Reply:
okay.  approved.